### PR TITLE
Add landing page after password reset

### PR DESCRIPTION
**Pull request by aanil**
- [x] Tests passing
- [x] Black formatting
- [ ] Migrations for any changes to the database schema
- [x] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,4 +17,5 @@ Please add a _short_ line describing the PR you make, if the PR implements a spe
 * Add support for database migrations using flask-migrate/Alembic ([#890](https://github.com/ScilifelabDataCentre/dds_web/pull/890))
 * Invite Researchers to projects ([911](https://github.com/ScilifelabDataCentre/dds_web/pull/911))
 * Changed `is_sensitive` to `non_sensitive` and set to default False ([#913](https://github.com/ScilifelabDataCentre/dds_web/pull/913))
-* Rearrangement and clean up of the token ([910](https://github.com/ScilifelabDataCentre/dds_web/pull/910))
\ No newline at end of file
+* Rearrangement and clean up of the token ([910](https://github.com/ScilifelabDataCentre/dds_web/pull/910))
+* Add landing page after password reset ([931](https://github.com/ScilifelabDataCentre/dds_web/pull/931))
```
Comment (inaod568) [inline CHANGELOG.md:21]: Put these in a new sprint

**Review by inaod568 — CHANGES_REQUESTED** [review]

**Comment by inaod568** [comment]
Looks good @aanil

**Review by alneberg — APPROVED** [review]
Looks good and works well! I think we might want to have slightly different page content for different types of users in the future but this sort of works for unit users as well.

File: CHANGELOG.md (hunk 1 above)
Comment (aanil) [reply CHANGELOG.md]: Done!

**Comment by inaod568** [comment]
> Looks good and works well! I think we might want to have slightly different page content for different types of users in the future but this sort of works for unit users as well.

Yes I think this works for now and we can improve on it

**Pull request merged**